cmd/report.go

```diff
@@ -45,7 +49,21 @@ func reportRun(cmd *cobra.Command, args []string) error {
 	}
 
 	if !dryRun {
-		if err = doReportCSVGenerate(ignoreArchived, allResults); err != nil {
+		filePath, err := cmd.Flags().GetString("file-path")
+		if err != nil {
+			return fmt.Errorf("%w", err)
+		}
+
+		matched, err := regexp.MatchString(`.*.csv`, filePath)
```

Comment:
Can we avoid doing this as it would restrict users to use only regular files for output and they may want to send it via a pipe to another process, like when you are downloading `*.tar.gz` and you can send it to `tar` command to decompress it without the need of saving it.

Reply:
OK I just wanted to put in place a safety for not allowing other paths such as \ or / - the file outputted is a csv so I figured I would make sure consumer added that as an extension.